themes/src/main/resources/theme/keycloak-preview/account/resources/app/keycloak-service/keycloak.service.ts

```diff
@@ -84,6 +84,21 @@ export class KeycloakService {
         return KeycloakService.keycloakAuth.realm;
     }
 
+    public get userName(): string {
+        let userName = 'Anonymous';
```

Comment:
@edewit This needs to be localized.

Reply:
I think we don't actually need to localize this. If I got it right, this won't be never displayed because KC always includes at least `preferred_username` in tokens, am I right? I understood this is just a failsafe of sort.

Reply:
I don't know enough to say with 100% certainty.  It's not a big deal to localize just in case.